projects/plugins/crm/modules/mailpoet/includes/class-mailpoet-background-sync-job.php

```diff
@@ -334,18 +329,17 @@ public function import_local_subscribers( $page_no = -1 ) {
 
 		}
 
-
 		// check for completion
 		$total_page_count = $this->get_total_page_count();
-		if ( $page_no >= $total_page_count ) {
+		if ( $page_no >= ( $total_page_count - 1 ) ) {
 
 			$this->debug( 'MailPoet subscriber import complete!' );
 
 			// we're at 100%, mark sync complete
 			$this->set_first_import_status( true );
 
-			// set pointer to last page
-			$this->set_resume_from_page( $total_page_count );
+			// set pointer to last processed page
+			$this->set_resume_from_page( $page_no );
```

Comment:
We have to set the latest processed page here, not the total, that is `total_pages - 1` or `$page_no`